src/syslog_mcp/tools/interface/utility_tools.py

```
@@ -0,0 +1,294 @@
+        # Analysis Layer - pure business logic
+        analysis_data = analyze_export_data(raw_logs, export_config)
+        
+        # For actual export, you would write the logs to a file here
+        # For now, just return a summary
+        
+        # Presentation Layer - pure formatting  
+        summary = format_export_summary(analysis_data)
+        
+        # Add export data in requested format
+        if format_type.lower() == "json" and raw_logs:
+            summary += f"\n## Export Data (First 5 records):\n```json\n"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)